src/main/scala/pt/tecnico/dsi/openstack/keystone/models/Role.scala

```diff
@@ -36,7 +36,19 @@ object Role {
     name: Option[String] = None,
     description: Option[String] = None,
   )
+
+  object Assignment {
+    implicit val decoder: Decoder[Assignment] = { cursor: HCursor =>
+      for {
+        // TODO: How to test if downField worked?
```

Comment:
Codacy found an issue: [Take the required action to fix the issue indicated by this comment.](https://app.codacy.com/gh/ist-dsi/scala-keystone-client/pullRequest?prid=5985393)